UIColorExtension.swift

```diff
@@ -0,0 +1,72 @@
+
+/*
+ * Jtribe
+ * UIColorExtension
+ *
+ * Description:
+ * Extensions to make dealing with instances of UIColor more simple, intuative and powerful.
+ *
+ * Jtribe Copyright © 2016
+ */
+
+import UIKit
+
+// MARK: Lightening & Darkening
+
+// Description:
+// This extension provides methods to lighten and darken a UIColor.
+
+extension UIColor {
+	
+	func lighten() -> UIColor {
+		return lightenByPercentage(0.2)
+	}
+	
+	func darken() -> UIColor {
+		return darkenByPercentage(0.2)
+	}
+	
+    func lightenByPercentage(percentage: CGFloat) -> UIColor {
```

Comment:
Also the function name could be `brightness(percentage p: CGFloat)` so you'd call it `.brightness(percentage: 20)` which would be clearer and "more Swift"